scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -280,28 +283,13 @@ def get_threshold_metrics(y_true, y_pred):
     model: pipeline.named_steps['classify']
     for model, pipeline in final_pipelines.items()
 }
-
-
-# In[22]:
-
-def get_coefficients(classifier, X_mat):
-    coef_df = pd.DataFrame.from_items([
-        ('feature', X_mat.columns),
-        ('weight', classifier.coef_[0]),
-    ])
-
-    coef_df['abs'] = coef_df['weight'].abs()
-    coef_df = coef_df.sort_values('abs', ascending=False)
-    
-    return coef_df
-
 coef_df_dict = {
```

Comment:
Let's make this a single df with a `model` column. Same with prediction dataframe. Simplifies output threefold when we have to store it.